Added functionality to switch between dev screen and client screen

DieEneKetel opened the pull request.

DieEneKetel committed:
Feat:Basic window & phantom text functionality

DieEneKetel committed:
Refactor: changed file structure to that of dev

DieEneKetel committed:
Merge branch 'dev' into Feature/DevScreen

DieEneKetel committed:
Feature: chosen client can be loaded in dev screen

DieEneKetel committed:
fix: completed code for challange 2 - lily and hid trees in gameScene

DieEneKetel committed:
Merge branch 'dev' into Feature/DevScreen

DieEneKetel committed:
feat: added coin increase on code written correctly

DieEneKetel committed:
Merge branch 'dev' into Feature/DevScreen

DieEneKetel committed:
Fix: wallet singleton path fix

Powerboy2015 reviewed (dismissed):
I lied, too many small bugs. Most features not working

Powerboy2015 committed:
Merge branch 'dev' into Feature/DevScreen

Powerboy2015 commented on Scenes/DevScene/DevScreen.cs:
```diff
@@ -0,0 +1,73 @@
+	{
+		ClientSettings.ChosenClient.CodingChallenges[1].PlayerCode = Input_Field.Text;
+
+		var ClientScreen = GD.Load<PackedScene>("res://Scenes/ClientScene/ClientScreen.tscn");
```
This created double instances because you're not deleting the other one , check `clientscreen.cs`

Powerboy2015 commented on Scenes/DevScene/DevScreen.cs:
```diff
@@ -0,0 +1,73 @@
+	
+	private void OnCloseWindowPressed()
+	{
+		ClientSettings.ChosenClient.CodingChallenges[1].PlayerCode = Input_Field.Text;
```
0-based indexing

Powerboy2015 commented on Scenes/DevScene/DevScreen.cs:
```diff
@@ -0,0 +1,73 @@
+		Input_Field = GetNode<TextEdit>("Panel/CodeSpace/Input_Field");
+
+		// get the challanges
+		var Challenge = ClientSettings.ChosenClient.CodingChallenges[1];
```
0-based indexing (again)

Powerboy2015 committed:
merge &bugfix

- updated lily & thomas challenge references
- merged coin count & close button so both work
- fixed devScreen 1 based indexing to 0 (???)
